androidApp/src/main/java/com/radityalabs/qualitymatter/androidApp/ui/Theme.kt

```diff
@@ -0,0 +1,47 @@
+package com.radityalabs.qualitymatter.androidApp.ui
+
+import androidx.compose.foundation.isSystemInDarkTheme
+import androidx.compose.material.MaterialTheme
+import androidx.compose.material.darkColors
+import androidx.compose.material.lightColors
+import androidx.compose.runtime.Composable
+
+private val DarkColorPalette = darkColors(
```

Comment:
shouldn't this under `color` class?

Reply:
Actually don't sure mas.
I just follow the Android studio template when creating the new compose project.

This is what I understand.
- Color file is for raw color (hex / rgb color)
- Theme file is for color palette and anything related to theme

Reply:
ah okay, we can segregate later on if needed.